src/main.cpp

```diff
@@ -3293,15 +3293,21 @@ bool static DisconnectTip(CValidationState &state) {
     // 0-confirmed or conflicted:
     for(const CTransaction &tx: block.vtx) {
         SyncWithWallets(tx, nullptr);
+        for(unsigned int pos = 0; pos < tx.GetVscCcOut().size(); ++pos)
```

Comment:
Instead of introducing a new signal, let's use the existing SyncVoidedCert (updating the name to SyncCertStatusUpdate).
To be able to update in the wallet the sc with the current certId and epoch we should also signal the following case:
* last existing top quality cert of an epoch is disconnected (this cert is already communicated as voided)
* the sidechain has a previous epoch and a cert for it
We need to communicate a sync event also for this prev epoch certificate with the status BWT_ON (in this way wallet will be able to update last sidechain cert id and epoch with this one).

Modify the voidedCertsMap in a set of objects (CCertificateStatusUpdateInfo) each containing:
* certHash
* epoch
* sidechainId
* enum {BWT_ON, BWT_OFF}

Drop notification of sc creation